docs/database/sql-server-component.md

```diff
@@ -9,15 +9,15 @@ ms.date: 12/01/2023
 
 In this article, you learn how to use the .NET Aspire SQL Server component. The `Aspire.Microsoft.Data.SqlClient` library:
 
-- Registers a scoped <xref:Microsoft.Data.SqlClient.SqlConnection?displayProperty=fullName> factory in the DI container for connecting Azure SQL, MS SQL database.
-- Automatically configures the following:
-  - Connection pooling to efficiently managed HTTP requests and database connections
-  - Automatic retries to increase app resiliency
-  - Health checks, logging and telemetry to improve app monitoring and diagnostics
+-   Registers a scoped <xref:Microsoft.Data.SqlClient.SqlConnection?displayProperty=fullName> factory in the DI container for connecting Azure SQL, MS SQL database.
+-   Automatically configures the following:
+    -   Connection pooling to efficiently managed HTTP requests and database connections
+    -   Automatic retries to increase app resiliency
+    -   Health checks, logging and telemetry to improve app monitoring and diagnostics
```

Comment:
Please avoid reformatting, as it's noisy (causing longer review cycles/churn) and doesn't add value to the content.

```suggestion
- Registers a scoped <xref:Microsoft.Data.SqlClient.SqlConnection?displayProperty=fullName> factory in the DI container for connecting Azure SQL, MS SQL database.
- Automatically configures the following:
  - Connection pooling to efficiently managed HTTP requests and database connections
  - Automatic retries to increase app resiliency
  - Health checks, logging and telemetry to improve app monitoring and diagnostics
```